### PR TITLE
Simplify statements

### PR DESCRIPTION
**Pull request by joelchen**

### CODE REVIEW

File: utils/meta.go [hunk 1]
```diff
@@ -25,9 +25,6 @@ func NewValue(t reflect.Type) (v reflect.Value) {
 func ToArray(value interface{}) (values []string) {
 	switch value := value.(type) {
 	case []string:
-		if len(value) == 1 && value[0] == "" {
```
Comment (jinzhu) [inline utils/meta.go]: why remove this? I think this is required?

Comment (joelchen) [inline utils/meta.go]: I tried a test on Go Playground (https://play.golang.org/p/jEOT60xVC-) by commenting these lines and found that they are redundant. Please correct me if I'm wrong.

Comment (jinzhu) [inline utils/meta.go]: This is required by select many, as html's select many widget will submit a array with a blank string when none data selected, in this case, we want a empty array, but not with a nil value.

 `[]string{""}` should return `[]string{}`, but not `[]string{nil}`

Comment (joelchen) [inline utils/meta.go]: You are right. The return becomes a nil slice which I did not see in Go Playground's output. How about replacing those lines with values = []string{} (https://play.golang.org/p/ANIBdnO3go), which I have just submitted for pull request?

**Comment by jinzhu** [comment]
Hi @joelchen 

Sounds great, btw, could you submit another pull request to clean up the commit history?

Thank you.

**Pull request closed without merging**